Wrappers/Python/cil/framework/framework.py

```diff
@@ -2816,15 +2971,42 @@ def subset(self, dimensions=None, **kw):
             return temp
 
     def get_slice(self,channel=None, vertical=None, horizontal_x=None, horizontal_y=None, force=False):
-        '''
-        Returns a new ImageData of a single slice of in the requested direction.
-        '''
-        try:
-            geometry_new = self.geometry.get_slice(channel=channel, vertical=vertical, horizontal_x=horizontal_x, horizontal_y=horizontal_y)
-        except ValueError:
-            if force:
-                geometry_new = None
-            else:
+        """
+        Returns a new ImageData containing a data sliced in the requested directions.
+        
+        Slices can be in any combination of:
+            <dimension label> = index
+            <dimension label> = slice(index0,indexn,step)
```

Comment:
Better with to doc an `Examples` section

Reply:
I have added examples, but I think this is best at the top. In these methods it's actually clearer than the 'parameters' section